catalog/datasets/wikidata-entities-streets.jsonld

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://www.wikidata.org#entities-streets",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Wikidata: straten in Nederland"
```

Comment:
Or should we also include Belgian streets (since our 'places' query searches both Dutch and Belgian places)?

Reply:
Streets are messy things, I am a little hesitant to make the mess bigger than necessary. Let's stick to Dutch streets first and maybe scale up later if things go well and we have tested that Belgian streets are well documented too in WD.

Reply:
👍🏻